MaxText/tests/moe_test.py

```diff
@@ -177,7 +180,7 @@ def setUp(self):
     )
     self.rng = jax.random.PRNGKey(42)
     quant = Fp8Quantization()
-    self.model = mlp_block(
+    self.model = linears.mlp_block(
```

Comment:
Nit: should this be the NNX version?

Reply:
It seems this one is added by @cgarciae to test Linen initialization? We could delete it if we don't need it.